docs/source/dev/buildoptions.rst

```diff
@@ -21,7 +24,41 @@ CMake Option                   Values          Description
 ============================== =============== ==================================================
 
 :sup:`1` *not yet implemented*
-:sup:`2` e.g. C++ keywords, currently disabled only for MSVC*
+
+:sup:`2` e.g. C++ keywords, currently disabled only for MSVC
+
+
+Shared or Static
+----------------
+
+By default, we will build as a static library (``libopenPMD.a`` or ``openPMD.lib``) and install also its headers.
+You can only build a static or a shared library (``libopenPMD.so`` or ``openPMD.dll``) at a time.
```

Comment:
I had to read this a few times to understand what you mean. There are really two sentences here - one about either static or shared - and one about the names of the shared library.

Reply:
I can put the brackets of the first sentence in the second one.

Reply:
Would make it easier to understand IMO.